lessons/beginners/and-or/index.md

```diff
@@ -18,40 +18,78 @@ které se hodí do podmínek:
     </tr>
     <tr>
         <td><code>and</code></td>
-        <td><code>True and False</code></td>
+        <td><code>True and False</code><br><code>2 < 3 and 5 < 3</code></td>
         <td>„a zároveň“</td>
     </tr>
     <tr>
         <td><code>or</code></td>
-        <td><code>True or False</code></td>
+        <td><code>True or False</code><br><code>2 < 3 or 5 < 3</code></td>
         <td>„a nebo“</td>
     </tr>
     <tr>
         <td><code>not</code></td>
-        <td><code>not False</code></td>
+        <td><code>not False</code><br><code>not 5 < 3</code</td>
         <td>„ne“</td>
     </tr>
 </table>
 
-Pro příklad použijeme `and` v tomto programu.
+Například, chceš-li zjistit, jestli je kterékoli z dvou čísel záporné,
+můžeš napsat:
+
+```python
+a = float(input("Zadej první stranu obdélníka: ")
+b = float(input("Zadej druhou stranu obdélníka: ")
+
+if a <= 0 or b <= 0:
+    print("Délka nemůže být záporná!")
+```
+
+> [warning] Falešní kamarádi
```

Comment:
tenhle výraz znám z výuky angličtiny (actually vs. aktuálně - false friends), ale v tomto kontextu mu úplně nerozumím. Budou mu rozumět čtenáři?

Reply:
Já považuju `and` a `or` za hodně podobný případ jako “false friends” v angličtině: `if a or b <= 0` neznamená „jestli je a nebo b menší než nula“.

Myslím, že když tomu někdo rozumět nebude, tak se nic moc nestane. Je to jen nadpis. Ale pro toho, kdo ten termín zná, to může být přínosná paralela.